titan-compiler/coder.lua

```diff
@@ -264,23 +264,27 @@ local function codewhile(ctx, node)
         tmpl = [[
             while($CEXP) {
                 $CBLK
+                $CHECKGC
             }
         ]]
     else
         tmpl = [[
             while(1) {
+                luaC_checkGC(L);
```

Comment:
Why do we need a luaC_checkGC() here and why isn't it also using $CHECKGC?

Reply:
@mascarenhas this is just a slip, right? I think it's the only pending issue before merging this PR.

Reply:
Yes, this was an oversight, this line should be removed, as there is already a $CHECKGC at the end of the loop.